# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-libs14/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gnome-libs-1.4.1.7/gnome-config.in	2001-03-05 23:17:43.000000000 +0100
+++ new/gnome-config.in	2002-05-26 00:24:18.000000000 +0200
@@ -19,7 +19,7 @@
 pkglibdir=${libdir}/@PACKAGE@
 pkgincludedir=${includedir}/@PACKAGE@
 MODULE_VERSION=@PACKAGE@-@VERSION@
-module_dirs="$libdir"
+module_dirs="$libdir /usr/lib"
 
 # stolen from autoconf
 if (echo "testing\c"; echo 1,2,3) | grep c >/dev/null; then
@@ -113,6 +113,9 @@
 modversion=false
 moddatadir=false
 
+the_libs=""
+the_flags="-I${includedir}"
+
 while test $# -gt 0; do
     case "$1" in
     -*=*) optarg=`echo "$1" | sed 's/[-_a-zA-Z0-9]*=//'` ;;
